src/actions.py

```diff
@@ -28,6 +28,12 @@
     "unfold": "unfold",
 }
 
+mod.list("cursorless_multiple_target_action", desc="Supported actions for cursorless multiple targets")
```

Comment:
I wonder if we want a better name here.  Because "swap" has multiple targets and isn't included, and others may not as well.  Maybe we just bail on a good name and do what you did extension-side?
```suggestion
mod.list("cursorless_move_bring_action", desc="Cursorless move or bring actions")
```